Add missing include for std::begin and std::end in SkParseColor.cpp

https://skia-review.googlesource.com/c/skia/+/533981

--- a/src/3rdparty/chromium/third_party/skia/src/utils/SkParseColor.cpp
+++ b/src/3rdparty/chromium/third_party/skia/src/utils/SkParseColor.cpp
@@ -8,6 +8,8 @@
 
 #include "include/utils/SkParse.h"
 
+#include <iterator>
+
 static constexpr const char* gColorNames[] = {
     "aliceblue",
     "antiquewhite",
